src/com/time/collaborate/Summer.java

```diff
@@ -0,0 +1,4 @@
+package com.time.collaborate;
+
+public class Summer {
+}
```

Comment:
Please add a random method (that compiles!)

Reply:
Added a call to the Summer class from Main